tap_linkedin_ads/schemas/campaign_groups.json

```diff
@@ -128,6 +128,36 @@
         "null",
         "string"
       ]
+    },
+    "total_budget":{
+      "type": "object",
```

Comment:
Can you please add null in the data type with an object?

Reply:
Added null type along with object.